db/migrate/20210805131818_add_annexs_to_conclusion_draft_review.rb

```diff
@@ -0,0 +1,10 @@
+class AddAnnexsToConclusionDraftReview < ActiveRecord::Migration[6.0]
+  def change
+    create_table :annexes do |t|
+      t.string :title, null: false
+      t.text :description, null: true
```

Comment:
```suggestion
      t.text :description
```
Solo porque es lo por defecto, en el resto no lo ponemos.

Reply:
okey